web/templates/topic_sidebar_actions.hbs

```diff
@@ -11,13 +11,22 @@
     {{#if development}}
         <li class="hidden-for-spectators">
             <div class="tabs-container">
-                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.MUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.MUTED}}" data-tippy-content="{{t 'Mute topic' }}" aria-label="{{t 'Mute topic' }}">
+                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.MUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.MUTED}}" data-tippy-content="{{t 'Mute' }}" aria-label="{{t 'Mute' }}">
                     <i class="zulip-icon zulip-icon-mute-new"></i>
                 </div>
-                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.UNMUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.UNMUTED}}" data-tippy-content="{{t 'Unmute topic' }}" aria-label="{{t 'Unmute topic' }}">
+                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.INHERIT)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.INHERIT}}" data-tippy-content="{{t 'Regular' }}" aria-label="{{t 'Regular' }}">
                     <i class="zulip-icon zulip-icon-unmute-new"></i>
                 </div>
-                <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.FOLLOWED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.FOLLOWED}}" data-tippy-content="{{t 'Follow topic' }}" aria-label="{{t 'Follow topic' }}">
+                {{#if stream_muted}}
+                    <div class="tab-option tippy-zulip-tooltip {{#if (eq visibility_policy all_visibility_policies.UNMUTED)}}selected-tab{{/if}}" data-visibility-policy="{{all_visibility_policies.UNMUTED}}" data-tippy-content="{{t 'Unmute' }}" aria-label="{{t 'Unmute' }}">
+                        <i class="zulip-icon zulip-icon-unmute-new"></i>
+                    </div>
+                {{else if (eq visibility_policy all_visibility_policies.UNMUTED)}}
+                    <div class="tab-option tippy-zulip-tooltip selected-tab" data-visibility-policy="{{all_visibility_policies.UNMUTED}}" data-tippy-content="{{t 'Unmute' }}" aria-label="{{t 'Unmute' }}">
```

Comment:
Maybe we can do this in a single `if` conditional? `if stream_muted or visibility_policy==UNMUTED`.
That would make is very clear in which cases we can possibly show the UNMUTE button (and also deduplicate the inside HTML)

Reply:
Hm, right. Made the changes.